test/units_tests.jl

```diff
@@ -1,68 +1,74 @@
-using Unitful#, UnitfulPlots
-using OrdinaryDiffEq, DiffEqBase
-
-const UNITS_TEST_ALGS = [Euler(),Midpoint(),RK4(),SSPRK104(),SSPRK22(),SSPRK33(),
-    SSPRK432(),BS3(),BS5(),DP5(),DP5Threaded(),DP8(),Feagin10(),Feagin12(),Feagin14(),
-    TanYam7(),Tsit5(),TsitPap8(),Vern6(),Vern7(),Vern8(),Vern9()]
+using OrdinaryDiffEq, RecursiveArrayTools, Unitful
+using LinearAlgebra
 
-f = (y,p,t) -> 0.5*y / 3.0u"s"
-u0 = 1.0u"N"
-prob = ODEProblem(f,u0,(0.0u"s",1.0u"s"))
+@testset "Algorithms" begin
+    algs = [Euler(),Midpoint(),Heun(),Ralston(),RK4(),SSPRK104(),SSPRK22(),SSPRK33(),
```

Comment:
can we keep the inside of the testsets not indented? I like to keep the tests Juno-runnable.

Reply:
Oh, I didn't know that the indentation affects your workflow with Juno. So it can be fixed by not indenting at the top level inside of the `@testset`s?

The main motivation for introducing separate testsets for each ODE problem was the error
```julia
LoadError: cannot define function f; it already has a value
```
due to which the tests currently fail (see https://travis-ci.org/JuliaDiffEq/OrdinaryDiffEqExtendedTests.jl/builds/444193108) - I thought `@testset`s would be the cleanest way to solve those scoping issues. (Actually, even with these changes running `test OrdinaryDiffEqExtendedTests` still fails due to errors in another test file.)

Reply:
>Oh, I didn't know that the indentation affects your workflow with Juno. So it can be fixed by not indenting at the top level inside of the @testsets?

Yeah, that's all it is. If you indent a block, Juno will by default run the whole block instead of letting you do everything line by line which is just annoying. It's fixed by just not indenting the highest level of the `@testset`. It's not a big deal but it is a difference.